lib/structures/CategoryChannel.js

```diff
@@ -15,23 +15,18 @@ const GuildChannel = require("./GuildChannel");
 * @prop {Number} position The position of the channel
 * @prop {Boolean} nsfw Whether the channel is an NSFW channel or not
 * @prop {Collection<PermissionOverwrite>} permissionOverwrites Collection of PermissionOverwrites in this channel
-* @prop {Collection<GuildChannel>?} channels A collection of guild channels that are part of this category
+* @prop {Array<GuildChannel>} channels A collection of guild channels that are part of this category
```

Comment:
With the changed return type to Array, `A *collection* of guild channels` is no longer an accurate description.